sdk/python/kfp_tekton/tekton.py

```diff
@@ -31,34 +32,64 @@ class AnySequencer(ContainerOp):
                 because the pipeline will generate a unique new name in case of conflicts.
 
         any: List of `Conditional` containerOps that deploy together with the `main`
-                containerOp.
+                containerOp, or the condtion that must meet to continue.
     """
     def __init__(self,
-                 any: List[dsl.ContainerOp],
+                 any: Iterable[Union[dsl.ContainerOp, ConditionOperator]],
                  name: str = None,):
 
         tasks_list = []
+        condition_list = []
         for cop in any:
-            cop_name = sanitize_k8s_name(cop.name)
-            tasks_list.append(cop_name)
+            if isinstance(cop, dsl.ContainerOp):
+                cop_name = sanitize_k8s_name(cop.name)
+                tasks_list.append(cop_name)
+            elif isinstance(cop, ConditionOperator):
+                condition_list.append(cop)
+
         task_list_str = ",".join(tasks_list)
+        arguments = [
+                    "--namespace",
+                    "$(context.pipelineRun.namespace)",
+                    "--prName",
+                    "$(context.pipelineRun.name)",
+                    "--taskList",
+                    task_list_str
+                ]
+        conditonArgs = processConditionArgs(condition_list)
+        arguments.extend(conditonArgs)
 
         super().__init__(
             name=name,
             image="dspipelines/any-sequencer:latest",
```

Comment:
Can we make this image name configurable? This way is easier to run some testing in our internal CI pipelines.